src/Console/DirectiveCommand.php

```diff
@@ -25,16 +29,151 @@ class DirectiveCommand extends LighthouseGeneratorCommand
      */
     protected $type = 'Directive';
 
+    /**
+     * The imports required by the various interfaces, if any.
+     *
+     * @var Collection
+     */
+    protected $imports;
+
     protected function getNameInput(): string
     {
-        return ucfirst($this->argument('name')).'Directive';
+        return ucfirst(trim($this->argument('name'))).'Directive';
     }
 
     protected function namespaceConfigKey(): string
     {
         return 'directives';
     }
 
+    /**
+     * Build the class with the given name.
+     *
+     * @param  string  $name
+     * @return string
+     *
+     * @throws \Illuminate\Contracts\Filesystem\FileNotFoundException
+     */
+    protected function buildClass($name)
```

Comment:
```suggestion
    protected function buildClass(string $name): string
```

Always use the strictest possible type hints.